src/utils/utils_cache_line.c

```diff
@@ -157,7 +157,7 @@ void set_cache_line_dirty(struct ocf_cache *cache, uint8_t start_bit,
 			 * If this is first dirty cline set dirty timestamp
 			 */
 			env_atomic64_cmpxchg(&req->core->runtime_meta->dirty_since,
```

Comment:
env_get_tick_count (acquiring monotonic high precission timestamp) is potentially slow. it would make sense to first read "dirty_since" and attempt cmpxchg only if it was 0

Reply:
I think I would rename this function to "env_get_time_ns", since this has quite different semantics than ticks.

Reply:
AD 1. I did the check before exchanging value.
AD 2. I think the names in this group of functions (time-related) are the same as in our POSIX-compatible `ocf_env` header. Of course we can discuss it.